webapp/portlet/src/main/webapp/vue-apps/vertical-menu/components/VerticalMenuApp.vue

```diff
@@ -29,6 +29,9 @@ export default {
     fullDisplayAllowed() {
       return this.$vuetify.breakpoint.width >= this.$vuetify.breakpoint.thresholds.lg;
     },
+    isLanguageArabic() {
```

Comment:
IMO, it will better to use eXo.env.portal.orientation to check the test orientation, then testing the laguage